client/me/purchases/manage-purchase/index.jsx

```diff
@@ -656,14 +656,18 @@ class ManagePurchase extends Component {
 				const { extraPageCount, numberOfIncludedPages } = difmTieredPurchaseDetails;
 				return (
 					<>
-						{ translate(
-							'A professionally built %(numberOfIncludedPages)d page website in 4 business days or less',
-							{
-								args: {
-									numberOfIncludedPages: numberOfIncludedPages,
-								},
-							}
-						) }{ ' ' }
+						{ numberOfIncludedPages === 1
```

Comment:
Just thinking about this, `numberOfIncludedPages` will always be 5, right?

https://github.com/Automattic/wp-calypso/blob/a36f47100c6d9e6f6933a781e1409188658354a1/client/lib/purchases/index.ts#L819